### DISCUSSION TITLE
text: 'Expected ">" but found "/"'

### DISCUSSION THREAD

**Discussion by liushunli** [umijs/umi#11700] [Q&A] [answered]
<img width="926" alt="1696120680316" src="https://github.com/umijs/umi/assets/12014406/0376f48f-b4ca-4207-a448-fc382191301a">

<img width="944" alt="1696120706066" src="https://github.com/umijs/umi/assets/12014406/ce464133-c790-46c4-87a9-3bbe5bc83cea">

为什么按照官方文档配置，这个组件引入就是有问题

**Comment by fz6m** [comment, marked as answer]
只有 tsx 文件可以编写 react 节点，需要改成 `app.tsx`

**Reply by liushunli** [reply]
thanks for your answer, I have resolved it 。
